cnd/src/network/mod.rs

```diff
@@ -728,16 +731,16 @@ async fn insert_state_for_bob<AL, BL, AA, BA, AH, BH, AI, BI, AT, BT, DB>(
     swap_request: Request<AL, BL, AA, BA, AI, BI>,
 ) -> anyhow::Result<()>
 where
-    AL: Send + 'static,
-    BL: Send + 'static,
-    AA: Send + 'static,
-    BA: Send + 'static,
-    AH: Send + 'static,
-    BH: Send + 'static,
-    AI: Send + 'static,
-    BI: Send + 'static,
-    AT: Send + 'static,
-    BT: Send + 'static,
+    AL: Clone + Send + Sync + 'static,
```

Comment:
Why do all of these have to be Clone suddenly? What is the type that you are actually cloning?
Putting a type constraint on this fn for what we are actually cloning would reduce the noise drastically. See line 745 here.

Reply:
Good catch! And very didactic of you to let me figure out the answer rather than telling me what to do :relaxed: 

I'm still not used to this new style (which is obviously superior) because the compiler directs you towards doing it the "wrong" way.

Reply:
> Why do all of these have to be Clone suddenly? What is the type that you are actually cloning?

Just for clarity, this is the bound that it was looking for:
```rust
bob::State<AL, BL, AA, BA, AH, BH, AI, BI, AT, BT>: Clone
```